spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/JdbcEnvironmentRepository.java

```diff
@@ -128,6 +128,23 @@ public Environment findOne(String application, String profile, String label) {
 					}
 				}
 			}
+			// get properties without profile, equivalent to foo.yml, application.yml
```

Comment:
This code looks the same as the block above, can we extract this into a method and just call that method with the sql statement?

Reply:
Done. 👌